core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -83,12 +85,18 @@ oppia.directive('tutorCard', [
 
             $scope.hintsExist = Boolean(oppiaPlayerService.getInteraction(
               $scope.activeCard.stateName).hints.length);
+
+            $scope.contentAudioTranslations =
```

Comment:
Yeah, forgot about supplemental card directive. Was just a small fix as all the audio controls themselves are in tutor card directive